source/Calamari.Common/Features/Processes/CommandLine.cs

```diff
@@ -104,7 +104,7 @@ public CommandLineInvocation Build()
             {
                 EnvironmentVars = environmentVariables,
                 OutputToLog = outputToLog,
-                WorkingDirectory = workingDirectory
+                WorkingDirectory = workingDirectory!
```

Comment:
`WorkingDirectory` is an odd one... it's not null on the `get`, but nullable on the `set`.
If the field is null, it will return `Environment.CurrentWorkingDirectory`.

I'm not entirely sure how to represent that.